packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -1172,6 +1172,11 @@ class DisplayOptionsStatusExample extends React.Component<{}> {
           optionChecker={AccessibilityInfo.isReduceMotionEnabled}
           notification={'reduceMotionChanged'}
         />
+        <DisplayOptionStatusExample
+          optionName={'Prefer Cross-Fade Transitions'}
+          optionChecker={AccessibilityInfo.prefersCrossFadeTransitions}
+          notification={'prefersCrossFadeTransitionsChanged'}
```

Comment:
Do we need to add `prefersCrossFadeTransitionsChanged` to `AccessibilityEventDefinitionsIOS` inside `AccessibilityInfo.js`?

Reply:
@makovkastar I'm not sure, UIKit has this bug that causes `UIAccessibilityPrefersCrossFadeTransitionsStatusDidChangeNotification` to not get triggered when  Prefers Cross-Fade Transitions changes, I reported [this bug](https://developer.apple.com/forums/thread/699257) on Apple Developer Forums a couple months ago but they haven't fixed this yet, that's also the reason why I've removed support for this here -> https://github.com/facebook/react-native/pull/34406/commits/71632b4aa4625c42a5913a16fd36e8d82a438f42.

`AccessibilityInfo.prefersCrossFadeTransitions` should be enough to fix https://github.com/facebook/react-native/issues/31484 though, at least while apple doesn't fix this bug on UIKit